backend/src/question/question.service.ts

```diff
@@ -221,4 +350,52 @@ export class QuestionService {
       return this.formatQuestionSummaries(orMatched);
     }
   }
+
+  //Cron jobs to invalidate the cache
+
```

Comment:
For code readability, can consider doing one main function like `invalidateQuestionCache` with the cron decorator, that calls all the sub-functions.